frontend/i18n-scripts/languages.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-export LANGUAGES=( 'ja' 'zh-cn' 'ko' 'fr')
+export LANGUAGES=( 'ja' 'zh-cn' 'ko' 'fr' 'es')
```

Comment:
Oops! I forgot to undo this action after the previous [i18n strings](https://github.com/openshift/console/pull/14502/files) upload to memsource portal.